website/docs/r/mysql_flexible_server.html.markdown

```diff
@@ -119,15 +119,19 @@ The following arguments are supported:
 
 * `private_dns_zone_id` - (Optional) The ID of the private DNS zone to create the MySQL Flexible Server. Changing this forces a new MySQL Flexible Server to be created.
 
-~> **NOTE:** The `private_dns_zone_id` is required when setting a `delegated_subnet_id`. The `azurerm_private_dns_zone` should end with suffix `.mysql.database.azure.com`.
+~> **Note:** The `private_dns_zone_id` is required when setting a `delegated_subnet_id`. The `azurerm_private_dns_zone` should end with suffix `.mysql.database.azure.com`.
+
+* `public_network_access` - (Optional) Whether approved public traffic is allowed through the firewall to this server. Defaults to `Enabled`.
```

Comment:
From @neil-yechenwei comment, we can remove the Defaults to enabled as that will be a breaking change.

```suggestion
* `public_network_access` - (Optional) Whether approved public traffic is allowed through the firewall to this server.
```